packages/terra-hookshot/src/HookshotContent.jsx

```diff
@@ -122,6 +136,24 @@ class HookshotContent extends React.Component {
     }
   }
 
+  enableContentResizeListener() {
+    if (!this.contentResizeListenerAdded) {
+      this.resizeObserver = new ResizeObserver((entries) => {
+        this.props.onContentResize(entries[0].contentRect);
+      });
+      this.resizeObserver.observe(this.contentNode);
+      this.contentResizeListenerAdded = true;
+    }
```

Comment:
I don't know if this can happen with this implementation, but with the resize observers I believe it's possible to enter an infinite resizing loop depending on how the content resizes when the bounding rect resizes. I've never seen it happen with the ResponsiveElement so I think it's unlikely it will happen here, just thought it might be worth mention. The resize observer might internally debounce slightly to avoid this